fr_FR/maintaining.md

```diff
@@ -1,3 +1,8 @@
+___
+**Note du traducteur**
+
+C'est la traduction du fichier [maintaining.md](https://github.com/sindresorhus/ava/blob/master/maintaining.md). Voici un [lien](https://github.com/sindresorhus/ava/compare/0f0cbe9b0d5a71609a8f5a7afcb412f2bf41ad91...master#diff-af20adbc8ab4842b04d1f5c7df6f563a) vers les différences avec le master de AVA (Si en cliquant sur le lien, vous ne trouvez pas le fichier `maintaining.md` parmi les fichiers modifiés, vous pouvez donc en déduire que la traduction est à jour).
+___
 # Maintenance
```

Comment:
@sindresorhus This can be as interesting for the reader who wants to know if the translation is up to date. I would point out any way all in the contributing.md #2